db/migrate/20190320140509_add_error_message_to_accounting_problem_loan_transactions.rb

```diff
@@ -0,0 +1,7 @@
+class AddErrorMessageToAccountingProblemLoanTransactions < ActiveRecord::Migration[5.2]
+  def change
+    add_column :accounting_problem_loan_transactions, :error_message, :string
+    execute "UPDATE accounting_problem_loan_transactions SET error_message = 'has_multiple_loans' WHERE error_message IS NULL;"
```

Comment:
used up/down for this. Is there a better way? I didn't want to use default: has_multiple_loans because that's not actually a default; it's just the correct value for existing rows.

Reply:
In general I will do anything that needs to operate on the data as a separate migration from schema changes.

Reply:
That said, the table gets wiped whenever the QB data is reset, so it would be reasonable to just not worry about it, this table will be frequently trashed and repopulated and it'll have the right data next time.

Reply:
ah ok

Reply:
I will split it out